harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
@bterlson according to https://github.com/tc39/test262/pull/453#discussion-diff-47529015, this forces my test262 tests to have a specific ordering. That seems like a footgun that applies to any test author, regardless of environment. (If that comment is wrong, however, please comment on that thread?)